charts/catalog/templates/apiserver-deployment.yaml

```diff
@@ -59,6 +59,8 @@ spec:
         - "{{ .Values.apiserver.verbosity }}"
         {{- if .Values.apiserver.tls.requestHeaderCA }}
         - --requestheader-client-ca-file=/var/run/kubernetes-service-catalog/requestheader-ca.crt
+        {{ else }}
+        - --authentication-skip-lookup=true
```

Comment:
Hmm, I think it's possible to use delegated authn/authz without passing the requestheader-ca when the correct RBAC bindings are in place. See #936 and #981